ext/date/php_date.stub.php

```diff
@@ -6,17 +6,17 @@ function strtotime(string $time, ?int $now = null): int|false {}
 
 function date(string $format, ?int $timestamp = null): string {}
 
-function idate(string $format, ?int $timestamp = null): int|false {}
+function idate(string $format, ?int $timestamp = null): int {}
```

Comment:
These changes should probably not been made due to ZEND_ASSERT being a bad option.